tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/inflow.tss

```diff
@@ -0,0 +1,35 @@
+timeseries scalar settingsfile: /perm/mo/mocm/prg/oslisflood/ecmwf/lisflood/tests/data/LF_ETRS89_UseCase/settings/inflow.xml_4233ecb3-06d1-4565-9927-9cfcbcbba68c.xml date: Wed Feb 17 16:00:59 2021
```

Comment:
We can delete the folder "tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12" before the push